_run/common-kind.mk

```diff
@@ -48,6 +50,25 @@ kind-cluster-create:
 	kubectl apply -f "$(INGRESS_CONFIG_PATH)"
 	"$(AKASH_ROOT)/script/setup-kind.sh"
 
+.PHONY: kind-cluster-calico-create
+kind-cluster-calico-create:
+	kind create cluster \
+		--config "$(KIND_CONFIG_CALICO)" \
+		--name "$(KIND_NAME)"
+	kubectl apply -f "$(CALICO_MANIFEST)"
+	kubectl -n kube-system set env daemonset/calico-node FELIX_IGNORELOOSERPF=true
+	# Calico needs to be managing networking before finishing setup
+	kubectl apply -f "$(INGRESS_CONFIG_PATH)"
```

Comment:
can all of this be pushed into `setup-kind.sh` so that you don't need `kind-configure-calico` make target?